src/sentry/deletions/defaults/group.py

```diff
@@ -131,48 +104,9 @@ def tenant_ids(self) -> Mapping[str, Any]:
     def chunk(self) -> bool:
         """This method is called to delete chunks of data. It returns a boolean to say
         if the deletion has completed and if it needs to be called again."""
-        if not options.get("deletions.nodestore.parallelization-task-enabled"):
```

Comment:
A Split view shows more clearly that we're only keeping the code from in the `else` block. Notice that I'm adding `self.delete_events_from_snuba()` which is a regression I introduced in the previous PR.

<img width="1604" height="362" alt="Image" src="https://github.com/user-attachments/assets/de233e29-34c4-4e14-b1bf-7d7524fbb916" />

Reply:
That's correct. It is handled as part of it. `snuba` was ambiguous here.

There's the nodestore (BigQuery) and then there's the eventstore (via Snuba).

Deletions in the new task:
https://github.com/getsentry/sentry/blob/b074cec795a8c6af79cb23c935c4bdc552fe73b5/src/sentry/deletions/tasks/nodestore.py#L116
https://github.com/getsentry/sentry/blob/b074cec795a8c6af79cb23c935c4bdc552fe73b5/src/sentry/deletions/tasks/nodestore.py#L131
https://github.com/getsentry/sentry/blob/b074cec795a8c6af79cb23c935c4bdc552fe73b5/src/sentry/deletions/tasks/nodestore.py#L193-L201

Even deleting from the nodestore requires querying Snuba for the event IDs, thus, deleting from both stores actually hits Snuba (which it is ridicolous but here we are)